src/main/java/net/dries007/tfc/client/GrassColorHandler.java

```diff
@@ -0,0 +1,170 @@
+package net.dries007.tfc.client;
+
+import java.awt.Color;
+import java.util.Random;
+
+import net.dries007.tfc.ConfigTFC;
+import net.dries007.tfc.util.calendar.CalendarTFC;
+import net.dries007.tfc.util.climate.ClimateTFC;
+import net.minecraft.block.state.IBlockState;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.MathHelper;
+import net.minecraft.world.ColorizerGrass;
+import net.minecraft.world.IBlockAccess;
+import net.minecraft.world.gen.NoiseGeneratorPerlin;
+
+
+public class GrassColorHandler 
+{
+    static NoiseGeneratorPerlin NOISE_GRASS = new NoiseGeneratorPerlin(new Random("NOISE_GRASS".hashCode()), 2);
+    
+	public static Color january, february, march, april, may,
+						june, july, august, september, october,
+						november, december;
+
+    static
+    {
+    	resetColors();
+    }
+    
+    public static void resetColors()
+    {
+    	july = parseColor(ConfigTFC.Client.GRASS_COLOR.seasonColorSummer);
+    	october = parseColor(ConfigTFC.Client.GRASS_COLOR.seasonColorAutumn);
+    	january = parseColor(ConfigTFC.Client.GRASS_COLOR.seasonColorWinter);
+    	april = parseColor(ConfigTFC.Client.GRASS_COLOR.seasonColorSpring);
+    	june = blendWithAlphas(april, july, 0.35, 0.65);
+    	september = blendWithAlphas(july, october, 0.35, 0.65);
+    	december = blendWithAlphas(october, january, 0.35, 0.65);
+    	march = blendWithAlphas(january, april, 0.35, 0.65);
+    	august = blendWithAlphas(july, october, 0.65, 0.35);
+    	november = blendWithAlphas(october, january, 0.65, 0.35);
+    	february = blendWithAlphas(january, april, 0.65, 0.35);
+    	may = blendWithAlphas(april, july, 0.65, 0.35);
+    }
+    
+    
+    public static Color parseColor(String s)
+    {
+    	String[] parts = s.split(",");
+    	int red = Integer.parseInt(parts[0].trim());
+    	int blue = Integer.parseInt(parts[1].trim());
+    	int green = Integer.parseInt(parts[2].trim());
+    	int alpha = Integer.parseInt(parts[3].trim());
+    	
+    	return new Color(red, blue, green, alpha);
```

Comment:
Blue and Green are switched.

Also, If you do the above changes to the config, you may call the constructor directly:

```java
new Color(int rgba, boolean hasalpha)
```

or, if you don't care about alpha:
```java
new Color(int rgb)
```